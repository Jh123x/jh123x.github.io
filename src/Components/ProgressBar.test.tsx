import * as React from "react";
import { describe, expect, test } from "@jest/globals";
import ProgressBar from "./ProgressBar";

describe("ProgressBar", () => {
  test("should render correctly", () => {
    expect(
      <ProgressBar title={"test"} width={10} text={"test_text"} />
    ).toMatchSnapshot();
  });
});
