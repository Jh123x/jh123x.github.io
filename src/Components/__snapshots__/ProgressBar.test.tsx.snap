// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar should render correctly 1`] = `
<ProgressBar
  text="test_text"
  title="test"
  width={10}
/>
`;
